Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+FROM busybox:1.36.0 as release
```

Comment:
Should we use the hash and put the version in a comment? https://hub.docker.com/layers/library/busybox/1.36.0/images/sha256-acaddd9ed544f7baf3373064064a51250b14cfe3ec604d65765a53da5958e5f5?context=explore